### PR TITLE
Add gh-pages CI

### PR DESCRIPTION
**Pull request by cmaureir**

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          python-version: '3.11'
+
+      - name: install python packages
+        run: |
+          cd generate_site/
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: execute py script
+        run: python generate.py
```
Comment (cmaureir) [inline .github/workflows/actions.yml:24]:
```suggestion
        run:  |
            cd generate_site/
            python generate.py
```

**Comment by Mariatta** [comment]
Seems like there's merge conflict.

**Comment by Mariatta** [comment]
With this CI, does it mean we don't have to commit the generated pages anymore?

**Comment by cmaureir** [comment]
> With this CI, does it mean we don't have to commit the generated pages anymore?

Exactly, if it works properly, we would not need to have the generated files. I'd try to make some testing in a non-so-busy tie :eyes:

**Comment by cmaureir** [comment]
Merging this in order to check if it's properly working

**Pull request merged**